### ISSUE TITLE
GPS in JollaOpas stopped working after update to 1.1.9.28/Eineheminlampi

### ISSUE THREAD

**Issue by jtts** [hsarkanen/JollaOpas#7]
After update to 1.1.9.28/Eineheminlampi, GPS in JollaOpas stopped working. It works OK with Jolla's Map app, so it’s probably not a system wide problem.

**Comment by hsarkanen** [comment]
I have updated my device to SFOS 1.1.9.28 but I cannot reproduce the GPS issue. I can see my location on map and I could use it as a starting point. Have you tried to switch positioning on and off from SFOS global Settings-System-Location menu? Are you testing with the latest Jolla's Map app version 1.1.9.2?

![jollaopas_sfos2](https://cloud.githubusercontent.com/assets/5443367/9888536/48f59c34-5bfe-11e5-8c29-cc9198af41c5.jpg)

**Comment by jtts** [comment]
Yes, I tested the latest version of Sailfish Maps (sailfish-maps-1.1.9.2-10.35.1.jolla.armv7hl). GPS worked fine with it.

I found the reason for my GPS problems: though GPS was enabled system-wide, for some reason it was disabled inside JollaOpas itself. Is this the default setting?

Anyway, in the future it might be good idea from a usability point of view that instead of failing silently, JollaOpas should disable the "Use current location" feature if GPS is off. Or alternatively show an error message, e.g. "Can't find location: GPS is off". Same goes for tracking location in the map view.
